sht3x/sht3x.h

```diff
@@ -95,42 +156,84 @@ int16_t sht3x_measure(void);
  * Temperature is returned in [degree Celsius], multiplied by 1000,
  * and relative humidity in [percent relative humidity], multiplied by 1000.
  *
- * @param temperature   the address for the result of the temperature
+ * @param[in]  addr the sensor address
+ * @param[out] temperature   the address for the result of the temperature
  * measurement
- * @param humidity      the address for the result of the relative humidity
+ * @param[out] humidity      the address for the result of the relative humidity
  * measurement
+ *
  * @return              0 if the command was successful, else an error code.
  */
-int16_t sht3x_read(int32_t* temperature, int32_t* humidity);
+int16_t sht3x_read(sht3x_i2c_addr_t addr, int32_t* temperature,
+                   int32_t* humidity);
 
 /**
  * Enable or disable the SHT's low power mode
  *
- * @param enable_low_power_mode 1 to enable low power mode, 0 to disable
+ * @param[in] enable_low_power_mode 1 to enable low power mode, 0 to disable
  */
 void sht3x_enable_low_power_mode(uint8_t enable_low_power_mode);
 
+/**
+ * Enable or disable the SHT's low power mode
+ *
+ * @param[in] mode power mode selector
+ */
+void sht3x_set_power_mode(sht3x_measurement_mode_t mode);
+
 /**
  * Read out the serial number
  *
- * @param serial    the address for the result of the serial number
+ * @param[in]  addr the sensor address
+ * @param[out] serial    the address for the result of the serial number
+ *
  * @return          0 if the command was successful, else an error code.
  */
-int16_t sht3x_read_serial(uint32_t* serial);
+int16_t sht3x_read_serial(sht3x_i2c_addr_t addr, uint32_t* serial);
 
 /**
  * Return the driver version
  *
+ * @param[in]  addr the sensor address
+ *
  * @return Driver version string
  */
-const char* sht3x_get_driver_version(void);
+const char* sht3x_get_driver_version(sht3x_i2c_addr_t addr);
```

Comment:
I don't think the address is needed for this function, since it just returns a static string.